### ISSUE TITLE
allow content-available for iOS notification

### ISSUE THREAD

**Issue by EyMaddis** [strongloop/loopback-component-push#96]
It is currently not possible to set `"content-available": 1` in a way that iOS can handle it correctly.

A possible fix would be something like this:

https://github.com/Maddis1337/loopback-component-push/blob/master/lib/providers/apns.js#L104-L107

**Comment by bajtos** [comment]
@Maddis1337 thank you for reporting the issue, could you please send your patch as a pull request? Preferably including a unit test.

**Comment by ffr200** [comment]
was the pull request ever created for this?

**Comment by Maddis1337** [comment]
No

**Comment by ffr200** [comment]
Is this still an issue?  Should this have a pull request?  I've been using 1.5.3 and am now questioning whether my silent push notifications are coming through properly

**Comment by EyMaddis** [comment]
They are probably not coming through.
I created a PR: #112

**Comment by danielrvt** [comment]
Hi guys, do you know if this is working now?

**Comment by superkhau** [comment]
@danielrvt Sorry for the late reply, waiting on @EyMaddis to write some test for the PR he sent to verify the fix at #112.

**Comment by superkhau** [comment]
The fix was abandoned, does someone want to submit another PR based on #112 to resolve this issue?
